cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/IRConstruction.qll

```diff
@@ -89,6 +167,26 @@ private module Cached {
   Expr getInstructionUnconvertedResultExpression(Instruction instruction) {
     result = getInstructionConvertedResultExpression(instruction).getUnconverted()
   }
+}
+
+import Cached
+
+cached
+private module Cached {
```

Comment:
Why should all these predicates be cached? I understand that `module Raw` needs to be cached because it's referred to from the third phase of the IR, but aren't the predicates in `module Cached` only referred to internally in the now-unified IR stage?

Reply:
I've removed all the `cached` annotations, and the `Cached` module itself. RA diffs are trivial. The inlining boundaries changed, but it's the same net amount of work with fewer unnecessarily cached predicates.